README.md

```diff
@@ -673,6 +674,41 @@ Additional options (configurable via `ConnectWithOpts`):
 * `ClusterDiscoveryTime` - time interval to ask server for updated address list (works on with `NodesGetFunctionName` set)
 * `NodesGetFunctionName` - server lua function name to call for getting address list
 
+## Tests
+
+You need to [install Tarantool](https://www.tarantool.io/en/download/) to run tests.
+See [Installation](#installation) section for requirements.
+
+To install test dependencies (like [tarantool/queue](https://github.com/tarantool/queue) module), run
+```bash
+./deps.sh
+```
+
+To run tests, call
+```bash
+go clean -testcache && go test ./... -v -p 1
+```
+Tests set up all required `tarantool` processes before run.
+
+To run subpackage tests, go to a subpackage folder and run the same command.
+```bash
+cd multi
+go clean -testcache && go test -v
+```
+
+Queue tests require [tarantool/queue](https://github.com/tarantool/queue) module, install it with `./deps.sh`.
+```bash
+cd queue
+./deps.sh
+go clean -testcache && go test -v
+```
+
+UUID tests require [Tarantool 2.4.1 or newer](https://github.com/tarantool/tarantool/commit/d68fc29246714eee505bc9bbcd84a02de17972c5).
+```bash
+cd UUID
```

Comment:
No such directory. Seems should be `uuid`.
```
sergeyb@pony:~/sources/go/src/github.com/tarantool/go-tarantool$ cd UUID
-bash: cd: UUID: No such file or directory
sergeyb@pony:~/sources/go/src/github.com/tarantool/go-tarantool$ stat uuid/
  File: uuid/
  Size: 4096            Blocks: 8          IO Block: 4096   directory
Device: 10305h/66309d   Inode: 4679017     Links: 2
Access: (0775/drwxrwxr-x)  Uid: ( 1001/ sergeyb)   Gid: ( 1001/ sergeyb)
Access: 2022-02-02 18:47:33.948968134 +0300
Modify: 2022-02-07 09:36:19.392040602 +0300
Change: 2022-02-07 09:36:19.392040602 +0300
 Birth: -
```

Reply:
Fixed